### PR TITLE
EM175: add opened_at to external pull request model

### PR DESCRIPTION
**Pull request by GuilleLeopold**
https://rootstrap.atlassian.net/jira/software/projects/EM/boards/42?selectedIssue=EM-175

### CODE REVIEW

**Review by santib — APPROVED** [review]

File: db/structure.sql [hunk 1]
```diff
@@ -47,7 +47,7 @@ CREATE TYPE public.lang AS ENUM (
     'android',
     'others',
     'unassigned',
-    'react_native'
+    'vuejs'
```
Comment (SandroDamilano) [inline db/structure.sql:50]: Why this change?

**Review by SandroDamilano — APPROVED** [review]
🥇

File: db/structure.sql (hunk 1 above)
Comment (GuilleLeopold) [reply db/structure.sql:50]: It seemed like it was deleted in the previous PR, dont know why. @santib commented on that on the previous PR. And the order changed probably is because postgres version or something like that

Comment (GuilleLeopold) [reply db/structure.sql:50]: i will check, it happened the same on the other PR
